tests/FSharp.Test.Utilities/FSharp.Test.Utilities.fsproj

```diff
@@ -39,6 +40,7 @@
     <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="$(MicrosoftCodeAnalysisWorkspacesCommonVersion)" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="$(MicrosoftCodeAnalysisCSharpVersion)" />
     <PackageReference Include="Microsoft.CodeAnalysis.Test.Resources.Proprietary" Version="$(MicrosoftCodeAnalysisTestResourcesProprietaryVersion)" />
-    <PackageReference Include="NUnit" Version="$(NUnitVersion)" />
+    <PackageReference Include="NUnit" Version="$(NUnitVersion)" /> <!-- TODO: This should be removed once all NUnit frameworks are migrated to xUnit -->
+    <PackageReference Include="FluentAssertions" Version="$(FluentAssertionsVersion)" />
```

Comment:
Out of curiosity, is `FluentAssertions` giving benefit to any of the current tests?

Reply:
I guess only a bit better assertion messages. The seem more readable to me.